packages/coordinator/src/screens/addAppointments/AddAppointmentsForm.tsx

```diff
@@ -36,16 +34,70 @@ export default function AddAppointmentsForm(props: AddAppointmentsFormProps) {
 
   const isButtonDisable = () => !(hospital && date && slots);
 
-  const onSave = () => {
+  /*
+   * This is a cheat added for Beilinson hospital.
+   * They asked to be able to add appointments, with specific hours, to the whole day in a single click.
+   */
+  const onAddWholeDay = () => {
+    if (!date || hospital !== Hospital.BEILINSON) {
+      return;
+    }
+
+    const weekdayHoursAndSlots = [
+      [8, 1],
```

Comment:
Since this is a web application, we can change it pretty easily.

Saving this in a server would add latency for fetching it.

I think I will keep it as is for now